cardy/.babelrc

```diff
@@ -0,0 +1,13 @@
+{
+  "presets": [
+    "env",
+    "react",
+    "stage-0"
```

Comment:
Not sure we should be using this and `env`?

Reply:
Looks like this was auto-created when I incorrectly added [babel-env](https://www.npmjs.com/package/babel-env) instead of babel-present-env. Have tidied up to be only what we require.